docs/apm/agent-configuration.asciidoc

```diff
@@ -0,0 +1,34 @@
+[role="xpack"]
+[[agent-configuration]]
+=== APM Agent configuration
+
+beta[] APM Agent configuration allows you to fine-tune your agent configuration directly in Kibana.
+Best of all, changes are automatically propagated to your APM agents so there's no need to redeploy.
+
+To get started, simply choose the service and environment you wish to configure.
+Agents will periodically poll APM Server and apply any configuration changes.
```

Comment:
could remind users here that APM Server acts as a proxy between agents and Kibana

Reply:
could mention (if you think is not obvious) that if no environment is set, configuration will only be propagated to agents that do not have environment